linodecli/configuration.py

```diff
@@ -554,6 +566,19 @@ def configure(self):
             "Please select a valid Image, or press Enter to skip",
         )
 
+        res = self._do_get_request("/profile/sshkeys", token=config["token"])
+        if 'data' in res:
+            config["authorized_users"] = self._default_thing_input(
+                "Do you want to use the following username as a authorized_user on New Linodes?",
+                [username],
+                "Default Option (Optional): ",
+                "Please select a valid Option, or press Enter to skip",
+            )
```

Comment:
It might add some flexibility to allow users to select any number of sub-users as defaults.
Something along the lines of:
- Get a list of users using [this endpoint](https://www.linode.com/docs/api/account/#users-list) 
- Prompt the user to select a user who should be given default access to newly-created Linodes
- If empty input:
  - Break out of the loop
- If user option is specified:
  - Append that user to `authorized_users` 
  - Drop that user from the user options
  - If the list of user options is now empty:
    - Break out of the loop 

Alternatively, we could simplify this to prompt whether the user wants to automatically be granted access to newly-created Linodes. (e.g. `Would you like to grant the current user ({}) access to all newly created Linodes? y/n`)

Reply:
Good catch I meant to clean this up and use that endpoint and totally forgot. I had some issues with writing and reading from the config file with lists so for now i think its best to stick to just one user, maybe we can expand on this in the future to use a list